src/arizona_tpl_scan.erl

```diff
@@ -30,8 +30,11 @@ Template scanner.
 %% API funtions.
 %% --------------------------------------------------------------------
 
+-spec string(String) -> Tokens
+    when String :: binary() | string(),
+         Tokens :: [arizona_tpl_parse:token()].
 string(Str) when is_binary(Str) ->
-    {ok, scan(Str, Str, 0, 0), 1};
+    scan(Str, Str, 0, 0);
```

Comment:
Got rid of all the `{ok, _` that weren't being used to signal "good behaviour" (i.e. had no `{error, _` counterpart, for example...